docs/content/users/quickstart.md

```diff
@@ -341,7 +341,74 @@ Here's a quickstart instructions for a number of different environments:
     
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
+    
+    ### Using Vite
+    The default Laravel Vite template will not work with DDEV. To make DDEV and Vite play nicely a few additionaly steps are required.
+
+    #### Setup a Vite server
+    Install the ddev-viteserve addon.
+    ```bash
+    ddev get torenware/ddev-viteserve
+    ```
+    In `.ddev/.env` change `VITE_PROJECT_DIR=frontend` to `VITE_PROJECT_DIR=.`
+
```

Comment:
Although it is marked as experimental, I think the following is important to add:

```md
In `.ddev/.env` add `VITE_JS_PACKAGE_MGR=npm`. This tells `ddev-viteserve` to use `npm` to manage packages, and 
helps keep dependencies in sync.
```

Laravel prefers `npm` which uses the `package.lock`. See https://github.com/torenware/ddev-viteserve/issues/3

Reply:
Thanks @tyler36 - can you try using the "Suggestion" feature when you know what is the best thing to do? You can just click the icon shown, it makes it so easy for the author of a PR then to just accept and commit your suggestion.

<img width="285" alt="Expanded_the_Laravel_quickstart_with_instructions_on_how_to_run_Vite_by_Ellarddekoeijer_·_Pull_Request__4236_·_drud_ddev" src="https://user-images.githubusercontent.com/112444/192920089-aa70a81b-ed8b-4421-99c7-04dc2f6e82f4.png">

Reply:
> Although it is marked as experimental, I think the following is important to add:
> 

I should probably change the docs; I no longer consider this to be experimental.

> ```lisp
> In `.ddev/.env` add `VITE_JS_PACKAGE_MGR=npm`. This tells `ddev-viteserve` to use `npm` to manage packages, and 
> helps keep dependencies in sync.
> ```

Does Laravel itself _rely_ on npm and package.lock? In general, I'd argue pnpm is a better package manager, especially in terms of its speed.
> 
> Laravel prefers `npm` which uses the `package.lock`. See [torenware/ddev-viteserve#3](https://github.com/torenware/ddev-viteserve/issues/3)

Does it?